README.MD

```diff
@@ -543,6 +580,8 @@ address:
 ```
 cat /var/lib/misc/dnsmasq.leases
 ```
+In some distros you don't have this file. In that case use `arp` in the terminal to do the same thing.
```

Comment:
You always have the dnsmasq.leases file but the location might differ. Also `arp -n` is probably the command to go with.